content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -86,10 +86,16 @@ def vm_prov_option_value(prov_option, options_array = [])
                :flavor        => flavor_obj(@miq_request.get_option(:instance_type)),
                :number_of_vms => get_option_value(@miq_request, :number_of_vms),
                :cloud         => vm_provision_cloud?}
+  # number_of_vms doesn't exist for VmReconfigureRequest
+  args_hash[:number_of_vms] = 1 if @reconfigure_request
```

Comment:
@tinaafitz 
Does this mean a reconfigure can only happen on a single vm and not a group of vms?